---
title: Robert's Research
permalink: /research
---
 
<h1>Overview</h1>
I work on problems at the intersection of biology and computing, my interests are particularly focused on questions about evolutionary theory and the way natural selection works. But I'm happy to advise projects with particularly motivated students that are outside of my particular specialty. If you are interested in getting a feel for the kinds of topics currently being investigated in computational biology you can come by the [Bioinformatics Reading Group](/BRG) or take a look at the schedule there. 

<h2>Main Research Topics</h2>

<h3>Measuring Selection in Natural Populations</h3>
The full extend to which selection acts on different parts of the genome is not fully understood. Which kinds of genes are important to selection? Which parts of non-coding DNA are important? Answering these questions requires large amounts of genomic data and computational analysis. 

I'm particularly interested in investigating the efficacy of machine learning methods to measure selection using population genetic data. The signals can be subtle and some kinds of selection are difficult to detect with traditional methods, these attributes make this a compelling problem for machine learning solutions.

**Students who are interested in this topic** and want to work with me might use available data (e.g. [https://www.ncbi.nlm.nih.gov/sra](https://www.ncbi.nlm.nih.gov/sra)) to answer some of these questions or developing new methods to investigate them.

Some of my previous work in this area: 
* [Williamson, et al. (2014) Evidence for Widespread Positive and Negative Selection in Coding and Conserved Noncoding Regions of *Capsella grandiflora*. PLoS Genetics.](https://doi.org/10.1371/journal.pgen.1004622)

<h3>Testing New Population Genetics Methods</h3>
New methods for measuring population genetic parameters, like selection or recombination, are being developed rapidly. Comparing these methods can be difficult, as many of them make different assumptions about the data or history of populations. One way of testing the efficacy of these methods is to use simulated data for which we know the "true" history of a population. I am interested in using simulation approaches like [SLiM](https://messerlab.org/slim/) to generate data and answer questions about the efficacy of classic and novel methods.

**Students who are interested in this topic** may chose a statistical test, method, or suite of methods for which we might want to test various assumptions.

Some of my previous work in this area:
* [Vanderpool, et al. (2020) Primate phylogenomics uncovers multiple rapid radiations and ancient interspecific introgression. PLoS Biology.](https://doi.org/10.1371/journal.pbio.3000954)
* I also work on projects in this area with my long term collaborator [Dr. Emily Josephs](https://plantbiology.natsci.msu.edu/directory/emily-josephs/).

<h3>Individual Based Simulations</h3>
Individual based models can be used to answer questions about the assumptions of models or how 'noisy' we should expect an experiment to be. These models explicitly simulate individuals and their behavior in some environment. For example, recently I worked on a simulation to help study how male fish chose the females they mate with, based on data about the female and other males in the environment. This method is very broadly applicable to many scales of questions.

**Students who are interested in this topic** should discuss with me the particular question they are interested in answering and consider what the key attributes of individuals they want to simulate are.

Some of my previous work in this area:
* [Daniel & Williamson. (2020) Males optimally balance selfish and kin-selected strategies of sexual competition in the guppy. Nature Ecology & Evolution.](https://doi.org/10.1038/s41559-020-1152-3)
* My RHIT undergrad thesis was an individual based model, if you can find it!

<h3>Machine Learning in Biology</h3>
I'm interested in applying machine learning approaches to questions in life sciences, both in genetics and other fields. This is a new area of research for me, so there is much room for me to grow. 

Currently in collaboration with Dr. Alfred Tuley in the RHIT Chemistry department I'm working on a project where we hope to use machine learning approaches to help predict which proteins a particular chemical "probe" binds to. The goal of this research is to help develop drugs that can target specific proteins.

**Students who are interested in this topic** should talk to me about the life science domain they want to apply ML methods to. 

<h2>Past Research Students</h2>

<h3>Mason Reid - BIO 2024 </h3>
"The Genetic History of Hemp and Drug Type *Cannabis* Since Domestication."

Mason is working on a project to do scans of selection on different types of *Cannabis* to detect the effects of domestication across the genome.

<h3>Vidhu Naik (CS) and Jackson Shen (CS) - 2023 </h3>
"Detecting Recent Frequency Dependent Selection using Random Forests and Bayesian Networks."

Vidhu and Jackson worked on a Senior Research Project to develop a pipeline to train machine learning algorithms to detect balancing selection. They then applied this pipeline to *Capsella grandiflora* data to compare to earlier approaches. 

<h3>Michael Hicks - BIO 2023 </h3>
"Assessing the Effects of the Sonic Hedgehog Pathway Inhibitor Genistein on the Growth and Regeneration of *H. littoralis*."

Michael worked to create protocols for testing the effects of one particular protein pathway on the regeneration abilities of Hydra. 

<h3>Andrea (Ziqi) Chen - CS and BIO minor 2024</h3>
"The effects of learning on the evolution of female mate choice." 

Andrea worked on this project through the RSURF program over the summer of her second year at Rose. This work will be presented at the Evolution 2023 conference. 

Andrea will be working on a thesis project in 2023-24.

<h3>Ferguson Zhang - BIO/BCMB 2021</h3>
"The genome-wide computational detection of short-term balancing selection in *Capsella grandiflora*."

Ferguson worked  on using deep sequencing of one population of *Capsella grandiflora* to detect signals of balancing selection across the genome.

<h3>Kathi Munoz-Hofman - BMTH 2020</h3>
"Using Support Vector Machines and CKSAAP to Identify Protein Targets of Halopyridines."

 Kathi worked on a project in collaboration with Dr. Alfred Tuley to identify which proteins a particular chemical probe would bind to. 